### PR TITLE
accept Path in load_assets_from_dbt_manifest

### PR DESCRIPTION
**Pull request by sryza**

### CODE REVIEW

File: python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py [hunk 1]
```diff
@@ -639,7 +640,7 @@ def load_assets_from_dbt_project(
 
 
 def load_assets_from_dbt_manifest(
-    manifest: Optional[Mapping[str, Any]] = None,
+    manifest: Optional[Union[Path, Mapping[str, Any]]] = None,
```
Comment (rexledesma) [inline python_modules/libraries/dagster-dbt/dagster_dbt/asset_defs.py:643]: Use https://docs.python.org/3/library/os.html#os.PathLike so we can support string paths as well
```suggestion
    manifest: Optional[Union[os.PathLike, Mapping[str, Any]]] = None,
```

**Review by rexledesma — APPROVED** [review]
Could also take in `str` if we use `os.PathLike`

**Comment by sryza** [comment]
> Could also take in str if we use os.PathLike

I'm seeing `isinstance("fdshkl", os.PathLike)` returns False. Is that what you'd expect?

**Comment by sryza** [comment]
Going to merge this as-is and we can do a followup to expand to PathLike for both this method and dbt_assets later if we choose

**Pull request merged**